projects/js-packages/base-styles/_buttons.scss

```diff
@@ -0,0 +1,93 @@
+@import './variables';
+@import './colors';
+
+.components-button {
```

Comment:
We discussed it with the team today, and discovered that these styles will also apply to non-RNA buttons, and in some cases could affect even non-Jetpack elements on the page.
We're discussing possible solutions in Slack: p1635967176436400-slack-CBG1CP4EN